sycl/include/CL/sycl/INTEL/esimd/esimd.hpp

```diff
@@ -498,6 +541,86 @@ ESIMD_INLINE simd<U, n> convert(simd<T, n> val) {
   return __builtin_convertvector(val.data(), vector_type_t<U, n>);
 }
 
+// ----------- Outlined implementations of esimd class APIs.
+
+template <typename T, int N> void simd<T, N>::copy_from(const T *const addr) {
+  constexpr unsigned Sz = sizeof(T) * N;
```

Comment:
Please, align style of variables naming: `addr` and `Sz` have different style.

Reply:
ok. Looks like we need  some update to the coding style, as SYCL public API spec uses lower case style, but we use LLVM style internally, sometimes those contradict.